circle.yml

```diff
@@ -8,8 +8,8 @@ machine:
     SCRIPT_PATH: ./ci/scripts
     CERTIFICATES_PATH: ./ci/cert
     PROFILE_PATH: ./ci/profile
-    GH_USER: flyve-mdm-bot
-    GH_REPO_SLUG: flyve-mdm/flyve-mdm-ios-inventory-agent
```

Comment:
Why you dont use `CIRCLE_PROJECT_USERNAME` and `CIRCLE_PROJECT_REPONAME`?

Reply:
no problem, I can do it

Reply:
changed!